pkg/repository/udmrepo/repo-option-consts.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2020 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package udmrepo
+
+const (
+	UNIFIED_REPO_OPTION_STORAGE_TYPE_S3    = "s3"
```

Comment:
all these kinds of const variables should be named with this style UnifiedRepoOptionStorageTypeS3 ?

and the length of these variables could be shortened, maybe too long?